tools/clang/lib/CodeGen/CGExprCXX.cpp

```diff
@@ -219,7 +219,7 @@ RValue CodeGenFunction::EmitCXXMemberOrOperatorMemberCallExpr(
       if (Base->getValueKind() != ExprValueKind::VK_RValue) {
         LValue LV = EmitLValue(Base);
         if (LV.isSimple()) {
-          This = EmitLValue(Base).getAddress();
+          This = LV.getAddress();
```

Comment:
This was creating a duplicate call to load the value (which was later optimized away).